### PR TITLE
[TW-1545] Admin overview refresh

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Updated the featured snippet to align with our guidelines.
* Updated the left nav title to "Overview".
* Relocated the onboarding checklist link higher on the page to improve visibility.
* Reduced the number of headings on the page, and combined content from multiple headings in some cases.
* Updated the overview page to touch upon the major topics in the "Administration" section of the LC docs.
* Minor cleanup in `onboarding-checklist.md`.

### CODE REVIEW

**Review by claireriley — APPROVED** [review]
LGTM! couple small notes to consider

File: src/pages/docs/administration/admin-overview.md [hunk 1]
```diff
@@ -16,27 +16,30 @@ contextual_links:
     url: "https://blog.postman.com/managing-team-members-and-roles-in-postman-just-got-easier/"
 ---
 
-Postman provides a variety of tools to enable and enhance collaboration within your team. Whether you're getting started with a new team or managing an existing one, you can customize your team's Postman experience to align with your organizational needs. As a [Team Admin or Super Admin](/docs/collaborating-in-postman/roles-and-permissions/#team-roles), you can use the following features to set up, manage, and secure your Postman team.
+Postman has a variety of tools you can use to set up, manage, and secure your Postman team. Whether you're setting up a new team or managing an existing one, you can customize your team's experience to align with your organization's goals.
```
Comment (nerminamiller-postman) [inline src/pages/docs/administration/admin-overview.md:19]:
```suggestion
Learn how to set up, manage, and secure your Postman team. Whether you're setting up a new team or managing an existing one, you can customize your team's experience to align with your organization's goals.
```

File: src/pages/docs/administration/admin-overview.md [hunk 2]
```diff
@@ -16,27 +16,30 @@ contextual_links:
...
 
-### Set up your Enterprise features
+You can [configure your team settings](/docs/administration/managing-your-team/team-settings/) to further customize the experience for both your Postman team and your API's consumers. This includes managing your team's public profile and making your team discoverable.
 
-If you're setting up an [Enterprise team](https://www.postman.com/pricing), you can learn more about all of the extra features available to you with [Postman Enterprise plans](/docs/administration/enterprise/about-postman-enterprise/). Enterprise features include advanced identity management, reports, a Private API Network, API governance and security controls, and features that enhance collaboration and administrative customization.
+If you're setting up an [Enterprise team](https://www.postman.com/pricing), you can learn more about the extra features available to you with [Postman Enterprise plans](/docs/administration/enterprise/about-postman-enterprise/). Enterprise features include advanced identity management, reports, a Private API Network, API governance and security controls, and features that enhance collaboration and administrative customization.
```
Comment (nerminamiller-postman) [inline src/pages/docs/administration/admin-overview.md:37]:
```suggestion
If you're setting up an [Enterprise team](https://www.postman.com/pricing), additional features are available to you with [Postman Enterprise plans](/docs/administration/enterprise/about-postman-enterprise/). Enterprise features include advanced identity management, reports, a Private API Network, API governance and security controls, and features that enhance collaboration and administration customization.
```

File: src/pages/docs/administration/admin-overview.md [hunk 3]
```diff
@@ -16,27 +16,30 @@ contextual_links:
...
 
-## Manage your Postman team
+## Secure your Postman team
 
-You can add and remove team members, Partners, and Guests in your [team dashboard](https://go.postman.co/settings/team/members). You can also update roles, adjust your team plan, or update your team settings as needed. To learn more, see [Manage your team](/docs/administration/managing-your-team/managing-your-team/).
+Depending on your [plan](https://www.postman.com/pricing), you can secure your Postman team's users by setting up [SSO](/docs/administration/sso/intro-sso/), [SCIM](/docs/administration/scim-provisioning/scim-provisioning-overview/), and [domain verification and capture](/docs/administration/domain-verification-and-capture/domain-capture-overview/). You can also [manage your team's API keys](/docs/administration/managing-your-team/managing-api-keys/), [use the Secret Scanner](/docs/administration/managing-your-team/secret-scanner/) to detect exposed secrets, and [access your team's audit logs](/docs/administration/managing-your-team/audit-logs/).
```
Comment (nerminamiller-postman) [inline src/pages/docs/administration/admin-overview.md:41]:
```suggestion
Depending on your [plan](https://www.postman.com/pricing), you can secure your Postman team's accounts by setting up [SSO](/docs/administration/sso/intro-sso/), [SCIM](/docs/administration/scim-provisioning/scim-provisioning-overview/), and [domain verification and capture](/docs/administration/domain-verification-and-capture/domain-capture-overview/). You can also [manage your team's API keys](/docs/administration/managing-your-team/managing-api-keys/), [use the Secret Scanner](/docs/administration/managing-your-team/secret-scanner/) to detect exposed secrets, and [access your team's audit logs](/docs/administration/managing-your-team/audit-logs/).
```

Comment (nerminamiller-postman) [reply src/pages/docs/administration/admin-overview.md:41]: Consider breaking up into bullets for easier scanning.

File: src/pages/docs/administration/admin-overview.md [hunk 4]
```diff
@@ -16,27 +16,30 @@ contextual_links:
...
 
-## Secure your Postman team
+## Admin FAQs
 
-You can secure your Postman team's users by setting up [SSO](/docs/administration/sso/intro-sso/), [SCIM](/docs/administration/scim-provisioning/scim-provisioning-overview/), and [domain verification and capture](/docs/administration/domain-verification-and-capture/domain-capture-overview/). You can also [manage your team's API keys](/docs/administration/managing-your-team/managing-api-keys/), use the [Secret Scanner](/docs/administration/managing-your-team/secret-scanner/) to detect exposed secrets, and access your team's [audit logs](/docs/administration/managing-your-team/audit-logs/) as needed.
+For frequently asked questions about Postman teams, see [Admin FAQs](/docs/administration/admin-faqs/).
```
Comment (nerminamiller-postman) [inline src/pages/docs/administration/admin-overview.md:45]:
```suggestion
To learn more about managing Postman teams, see [Admin FAQs](/docs/administration/admin-faqs/).
```

File: src/pages/docs/administration/onboarding-checklist.md [hunk 5]
```diff
@@ -88,7 +88,7 @@ After you configure the Postman Enterprise app in your organization, it's recomm
 
 ## Set up your Postman team
 
-After you [set up Postman](#set-up-postman), enable your teammates to find and join your team. You can also share invites with them directly. Once people are added to your team, you can begin assigning them roles based on the permissions each user needs in your team. Then you can begin collaborating with your teammates in workspaces, collections, and other workspaces resources.
+After you [set up Postman](#set-up-postman-in-your-organization), enable your teammates to find and join your team. You can also share invites with them directly. Once people are added to your team, you can begin assigning them roles based on the permissions each user needs in your team. Then you can begin collaborating with your teammates in workspaces, collections, and other workspace resources.
```
Comment (nerminamiller-postman) [inline src/pages/docs/administration/onboarding-checklist.md:91]:
```suggestion
After you [set up Postman](#set-up-postman-in-your-organization), enable your teammates to find and join your team. You can also share invites with them directly. Once you add members to your team, you can begin assigning them roles based on their permissions. Then you can begin collaborating with your teammates in workspaces, collections, and other workspace resources.
```

**Review by nerminamiller-postman — APPROVED** [review]
Nice work, @MatthewPinsker-postman! I left you just a few small suggestions.

File: src/pages/docs/administration/admin-overview.md [hunk 6]
```diff
@@ -16,27 +16,30 @@ contextual_links:
...
-* [Manage your Postman team](#manage-your-postman-team)
+* [Set up and manage your Postman team](#set-up-and-manage-your-postman-team)
 * [Secure your Postman team](#secure-your-postman-team)
+* [Admin FAQs](#admin-faqs)
 
-## Set up your Postman team
+## Set up and manage your Postman team
 
-If you're setting up a new Postman team, check out Postman's [Onboarding checklist](/docs/administration/onboarding-checklist/).
+You can [invite and manage collaborators](/docs/administration/managing-your-team/managing-your-team/#managing-team-members) in your Postman team, which includes updating roles assigned to collaborators. You can also [adjust your team's Postman plan](/docs/administration/managing-your-team/managing-your-team/#managing-your-teams-postman-plan) as needed.
```
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/admin-overview.md]: I agree. Thanks!

[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/admin-overview.md]: Good idea. I added a note at the end of the section.

File: src/pages/docs/administration/admin-overview.md (hunk 2 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/admin-overview.md]: Thanks! I made updates based on your feedback.

File: src/pages/docs/administration/admin-overview.md (hunk 3 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/admin-overview.md]: @nerminamiller-postman I updated the section to use bullet points instead. Please let me know what you think. Thanks!

File: src/pages/docs/administration/admin-overview.md [hunk 7]
```diff
@@ -16,27 +16,30 @@ contextual_links:
     url: "https://blog.postman.com/managing-team-members-and-roles-in-postman-just-got-easier/"
 ---
 
-Postman provides a variety of tools to enable and enhance collaboration within your team. Whether you're getting started with a new team or managing an existing one, you can customize your team's Postman experience to align with your organizational needs. As a [Team Admin or Super Admin](/docs/collaborating-in-postman/roles-and-permissions/#team-roles), you can use the following features to set up, manage, and secure your Postman team.
+Postman has a variety of tools you can use to set up, manage, and secure your Postman team. Whether you're setting up a new team or managing an existing one, you can customize your team's experience to align with your organization's goals.
+
+You must be a [Team Admin or Super Admin](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) to set up, manage, and secure your Postman team.
+
+If you're setting up a new Postman team, you can check out Postman's [onboarding checklist](/docs/administration/onboarding-checklist/) to walk you through setting up your team.
```
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/admin-overview.md]: Thanks! I made updates based on your feedback.

File: src/pages/docs/administration/admin-overview.md (hunk 3 above)
Comment (nerminamiller-postman) [reply src/pages/docs/administration/admin-overview.md]: Looks great!

File: src/pages/docs/administration/admin-overview.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Administration overview"
-updated: 2023-10-27
+updated: 2023-12-19
```
Comment (jkonrath-postman) [inline src/pages/docs/administration/admin-overview.md:3]: Bump that up to 01-09

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**